dask/bytes/core.py

```diff
@@ -10,15 +13,15 @@
 
 
 def read_bytes(
-    urlpath,
-    delimiter=None,
-    not_zero=False,
-    blocksize="128 MiB",
-    sample="10 kiB",
-    compression=None,
-    include_path=False,
-    **kwargs,
-):
+    urlpath: str | list[str],
+    delimiter: Any | None = None,
+    not_zero: bool = False,
+    blocksize: str | int | None = "128 MiB",
+    sample: int | str | bool = "10 kiB",
+    compression: str | None = None,
+    include_path: bool = False,
+    **kwargs: Any,
+) -> tuple[Any, ...]:
```

Comment:
This could be covered with an `@overload` covering different values for `include_path`, but I leave it to you whether it;s worth your time :)

Reply:
Ah thanks for pointing that out, I think I got something that should work